environments/qa/hiera/common.json

```diff
@@ -16,28 +36,35 @@
         "127.0.0.2"
     ],
 
+    "grasshopper::setup::tenant_appdisplayname_escaped": "My+Timetable+-+Beta",
+
     /* Grasshopper repository */
     "app_ui_path": "/opt/grasshopper-ui",
     "app_install_method": "git",
     "app_install_config": {
         "source": "https://github.com/CUL-DigitalServices/grasshopper.git",
-        "revision": "master"
+        "revision": "ccf099550f74dded246c2cee9cb91d1cc30e78d8"
```

Comment:
The idea of this *QA* server, rather than a dev or nightly server is that it is a known release that we have chosen because of its readiness for QA. The intention would be to update this commit once every couple of days when new features or bugfixes become ready, but this would be a human decision, rather than automatic. This allows us (in particular Oszkar) to control what customers are seeing and move them forward in a controlled way. Having the specific commit recorded means that bug reports / QA results can be matched up against a specific codebase. As I say, there could be a separate dev/nightly server for the purposes of showing what the latest master is doing.